src/chaos/utils.go

```diff
@@ -0,0 +1,17 @@
+package chaos
+
+import (
+	"time"
+)
+
+// Wait cluster to reconfigure after chaos experiment
+func ExperimentSleep(durationStr string) error {
+	experimentDuration, err := time.ParseDuration(durationStr)
+	if err != nil {
+		return err
+	}
+
+	time.Sleep(experimentDuration)
+
+	return nil
+}
```

Comment:
Maybe we should just name this function Sleep?

Reply:
Yep